app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
I'm not  saying that I want to avoid using `create` as a method name. I think `create` is the correct name. I'm trying to avoid defining the `create` method until it's needed. I'm planning on trying to use `build` in all the places where I put the 'temporary fix' comment. I don't think I want to raise errors. I want to ask `person.valid?` and return `person.errors` if `valid?` is false